### ISSUE TITLE
Filepath not allowing path to subdirectory

### ISSUE THREAD

**Issue by SylvainCorlay** [jupyterhub/binderhub#456]
- not specifying a filepath starts with the root directory, as expected.
 - specifying a full path to a notebook behaves as expected.

However, we cannot specify a path to a subdirectory (where the notebooks would typically be).

**Comment by choldgraf** [comment]
Hmmm - I just tried this URL:

https://mybinder.org/v2/gh/binder-examples/zero-to-binder/master?filepath=binder

and it **did** put me in the `binder/` folder.

@SylvainCorlay can you share the link you're using that isn't working?

**Comment by betatim** [comment]
I think the problem is
```
https://mybinder.org/v2/gh/binder-examples/zero-to-binder/master?filepath=binder
```
vs
```
https://mybinder.org/v2/gh/binder-examples/zero-to-binder/master?filepath=binder/
```
(trailing slash). Should we sanitise that or just declare it illegal?

**Comment by choldgraf** [comment]
IMO we should sanitize something like that as it seems like a pretty reasonable assumption that users will make

**Comment by choldgraf** [comment]
@SylvainCorlay can you confirm that it works w/o the trailing slash?

**Comment by choldgraf** [comment]
I think this has now been fixed, #495 is closed as well. Please re-open if there is more to be done!
